factory/ideas.php

```diff
@@ -172,16 +190,16 @@ public function get_ideas($number = 10, $sort = 'date', $sort_direction = 'DESC'
 			// YEEEEEEEEAAAAAAAAAAAAAHHHHHHH
 			// From http://evanmiller.org/how-not-to-sort-by-average-rating.html
 			$sql = 'SELECT *,
-					((idea_votes_up + 1.9208) / (idea_votes_up + idea_votes_down) -
-	                1.96 * SQRT((idea_votes_up * idea_votes_down) / (idea_votes_up + idea_votes_down) + 0.9604) /
-	                (idea_votes_up + idea_votes_down)) / (1 + 3.8416 / (idea_votes_up + idea_votes_down))
-	                AS ci_lower_bound
-       			FROM ' . $this->table_ideas . "
-       			WHERE $where
+				((idea_votes_up + 1.9208) / (idea_votes_up + idea_votes_down) -
+	            1.96 * SQRT((idea_votes_up * idea_votes_down) / (idea_votes_up + idea_votes_down) + 0.9604) /
+	            (idea_votes_up + idea_votes_down)) / (1 + 3.8416 / (idea_votes_up + idea_votes_down))
+	            AS ci_lower_bound
+       				FROM ' . $this->table_ideas . "
+       				WHERE $where
```

Comment:
I doubt doing this in PHP has any advantages. The point of the query as I read it is to get just the top 5 most highly rated ideas only. To do so in PHP would mean getting every single idea, then doing the math and reducing the selection to the top five. As ugly as it looks, the math (or maths) seems rather basic.